fastapi/dependencies/utils.py

```diff
@@ -736,6 +737,10 @@ def get_schema_compatible_field(*, field: ModelField) -> ModelField:
     return out_field
 
 
+def is_form_data(field: Type[params.Body]) -> bool:
+    return field == params.Form or field == params.File or field == bytes
```

Comment:
What do you think about this?
```suggestion
    return field in (params.Form, params.File, bytes)
```

Reply:
Looks good to me, will commit this suggestion. 👍